backend/handlers/handlers_test.go

```diff
@@ -15,27 +15,48 @@
 package handlers_test
 
 import (
+	"bytes"
 	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/digitalbitbox/bitbox-wallet-app/backend"
 	"github.com/digitalbitbox/bitbox-wallet-app/backend/arguments"
+	"github.com/digitalbitbox/bitbox-wallet-app/backend/devices/usb"
 	"github.com/digitalbitbox/bitbox-wallet-app/backend/handlers"
+	"github.com/digitalbitbox/bitbox-wallet-app/util/system"
 	"github.com/digitalbitbox/bitbox-wallet-app/util/test"
 	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/require"
 )
 
+// testEnvironment implements backend.Environment
+type testEnvironment struct {
+}
+
+// NotifyUser implements backend.Environment
+func (testEnvironment) NotifyUser(text string) {
+}
+
+// DeviceInfos implements backend.Environment
+func (testEnvironment) DeviceInfos() []usb.DeviceInfo {
+	return usb.DeviceInfos()
```

Comment:
same here and the one below, you don't want to actually call anything here. At most, you could check input and mock output, but in theory the handlers unit test would not exercise this code as it is part of backend.go, not handlers.go